### PR TITLE
i836 style catalog

### PR DESCRIPTION
**Pull request by NateWr**
This commit styles each of the category list pages: catalog, category, series, new releases and search. It addresses [this issue](https://github.com/pkp/pkp-lib/issues/836).

### CODE REVIEW

File: templates/frontend/components/breadcrumbs_catalog.tpl [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+		<li>
+			<a href="{url router=$smarty.const.ROUTE_PAGE page="catalog"}">
+				{translate key="navigation.catalog"}
+			</a>
+			<span class="separator">{translate key="navigation.breadcrumbSeparator"}</span>
+		</li>
+		{if $parent}
+			<li>
+				<a href="{url op=$type path=$parent->getPath()}">
+					{$parent->getLocalizedTitle()}
```
Comment (asmecher) [inline templates/frontend/components/breadcrumbs_catalog.tpl]: Needs `|escape`

File: templates/frontend/components/breadcrumbs_catalog.tpl [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+				</a>
+				<span class="separator">{translate key="navigation.breadcrumbSeparator"}</span>
+			</li>
+		{/if}
+		<li class="current">
+			<h1>
+				{if $currentTitleKey}
+					{translate key=$currentTitleKey}
+				{else}
+					{$currentTitle}
```
Comment (asmecher) [inline templates/frontend/components/breadcrumbs_catalog.tpl]: Needs `|escape`

File: templates/frontend/components/monographList.tpl [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+	{if $featured && count($featured) > 0}
+		{foreach from=$featured key=id item=array_key}
+			{include file="frontend/objects/monograph_summary.tpl" monograph=$monographs[$id] isFeatured=1}
+		{/foreach}
+	{/if}
+
+	{assign var=counter value=1}
+	{foreach name="monographListLoop" from=$monographs item=monograph key=key}
+		{if is_array($featured) && array_key_exists($key, $featured)}
+			{php}continue;{/php}
```
Comment (asmecher) [inline templates/frontend/components/monographList.tpl]: There is a Smarty `{continue}` -- though I'm not sure it's in the version we use. Mind testing?

**Comment by NateWr** [comment]
@asmecher Good catch. I've added the escaping. The `{continue}` tag doesn't work in our version. I've tried in the past.

**Comment by asmecher** [comment]
Thanks; perhaps a good reason to dust off the migration to Smarty 3.0 that I tinkered with a while back:
http://pkp.sfu.ca/bugzilla/show_bug.cgi?id=8621

File: templates/frontend/objects/monograph_summary.tpl [hunk 4]
```diff
@@ -8,11 +8,17 @@
...
 	<a href="{url page="catalog" op="book" path=$monograph->getId()}" class="cover">
 		<img alt="{translate key="catalog.coverImageTitle" monographTitle=$monograph->getLocalizedFullTitle()|strip_tags|escape}" src="{url router=$smarty.const.ROUTE_COMPONENT component="submission.CoverHandler" op="thumbnail" submissionId=$monograph->getId() random=$monograph->getId()|uniqid}" />
 	</a>
+	{* {if $isFeatured}
+		<div class="featured">
+			Featured
+		</div>
+	{/if} *}
 	<a href="{url router=$smarty.const.ROUTE_PAGE page="catalog" op="book" path=$monograph->getId()}" class="title">
 		{$monograph->getLocalizedFullTitle()}
```
Comment (asmecher) [inline templates/frontend/objects/monograph_summary.tpl]: This also needs `|escape`. It's probably worth going through the front-end templates in general.

**Comment by NateWr** [comment]
@asmecher There's definitely been a few times it's tripped me up, finding docs and advice for features we don't have. But to be honest there are always workarounds. Probably not a high-priority thing at the moment.

**Comment by asmecher** [comment]
@NateWr, this needed a rebase, but I cherry-picked all the commits (minus the submodule commit) instead.

**Pull request closed without merging**